changelog.md

```diff
@@ -6,7 +6,8 @@
 
 ## New features
 
-* Add gds.graph.project.remoteDb endpoint for projecting graphs from AuraDB instances
+* Add the new concept of GDS Sessions, used to manage GDS computations in Aura, based on data from an AuraDB instance.
+  * Add `gds.graph.project.remote` endpoint for projecting graphs from AuraDB instances
```

Comment:
```suggestion
  * Add a new `gds.graph.project` endpoint to project graphs from AuraDB instances to GDS sessions.
```

Actually from the client perspective, its not called project.remote anymore but only just `gds.graph.project`.

Reply:
you're right of course!